Cython/Build/Dependencies.py

```diff
@@ -488,6 +490,13 @@ def parse_dependencies(source_filename):
         cimport_from, cimport_list, extern, include = m.groups()
         if cimport_from:
             cimports.append(cimport_from)
+            m_after_from = dependency_after_from_regex.search(source[m.end():])
```

Comment:
`regex.search()` accepts a start position as second argument. That way, we can avoid to slice and copy the source string here.

Reply:
Thanks for the suggestion - didn't know about that. Changed it in a new fixup.